bedevere/close_pr.py

```diff
@@ -0,0 +1,28 @@
+"""Automatically close PR that tries to merge maintenance branch into master"""
+import re
+
+import gidgethub.routing
+
+PYTHON_MAINT_BRANCH_RE = re.compile(r'^python:\d+.\d+$')
+
+router = gidgethub.routing.Router()
+
+@router.register("pull_request", action="opened")
+@router.register("pull_request", action="synchronize")
+async def close_invalid_pr(event, gh, *args, **kwargs):
+    """
```

Comment:
Drop the newline from the opening docstring (for style consistency in the codebase).